src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -309,6 +363,47 @@ public Jwt parse(String jwt) throws ExpiredJwtException, MalformedJwtException,
         }
     }
 
+    private void validateExpectedClaims(Header header, Claims claims) {
+        for (String expectedClaimName : expectedClaims.keySet()) {
+            Object expectedClaimValue;
+            Object actualClaimValue;
+
+            // since issued at is a date, call the specific method
+            // other methods deal with strings and the more
+            // general method can be used
```

Comment:
Similar to the above, there's a check to see if it's an `issedAt` check. All other types are handled in the `else`